content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config_download.md

```diff
@@ -15,7 +15,7 @@ kubeadm alpha kubelet config download [flags]
 ```
   # Download the kubelet configuration from the ConfigMap in the cluster. Autodetect the kubelet version.
   kubeadm alpha phase kubelet config download
-  
+
```

Comment:
@furukawa3 , I would not bother to clean up this file and all files found under the directory:
en/docs/reference/setup-tools/kubeadm/generated/
These files are auto-generated.

Reply:
I see!!!  Thanks.